stratum/hal/bin/tdi/BUILD

```diff
@@ -0,0 +1,155 @@
+# Copyright 2018 Google LLC
+# Copyright 2018-present Open Networking Foundation
+# Copyright 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+load("@rules_pkg//:pkg.bzl", "pkg_deb", "pkg_tar")
+load("//bazel:rules.bzl", "HOST_ARCHES", "stratum_cc_binary")
+
+licenses(["notice"])  # Apache v2
+
+package(
+    default_visibility = ["//visibility:public"],
+)
+
+stratum_bf_common_deps = [
+    "//stratum/glue:init_google",
+    "//stratum/glue:logging",
+    "//stratum/hal/lib/common:hal",
+    "//stratum/hal/lib/phal:phal_sim",
+    "//stratum/hal/lib/phal",
+    "//stratum/lib/security:auth_policy_checker",
+    "//stratum/lib/security:credentials_manager",
+    "@com_github_google_glog//:glog",
+]
+
+stratum_cc_binary(
+    name = "tdi_pipeline_builder",
+    srcs = [
+        "tdi_pipeline_builder.cc",
+    ],
+    arches = HOST_ARCHES,
+    deps = [
+        "//stratum/glue:init_google",
+        "//stratum/glue:logging",
+        "//stratum/glue/status",
+        "//stratum/glue/status:status_macros",
+        "//stratum/hal/lib/tdi:bf_cc_proto",
+        "//stratum/lib:utils",
+        "@com_github_nlohmann_json//:json",
+    ],
+)
+
+# SDE_INSTALL_TAR is not defined
+# no such target '@local_barefoot_bin//::sde_version_9.2.0'
+stratum_cc_binary(
+    name = "stratum_tdi",
+    srcs = [
+        "main_bfrt.cc",
+    ],
+    arches = HOST_ARCHES,
+    data = [
+        "tofino_skip_p4.conf",
```

Comment:
Please add this file also in addition to the no_bsp version.

Reply:
This is messy. TDI has a modified `tofino_skip_p4_no_bsp.conf` that appears to be DPDK-specific and no `tofino_skip_p4.conf`.

For now, I'm going to copy the files from the `barefoot` target. The build will require additional work to make it multi-target.